model/device.go

```diff
@@ -33,12 +33,15 @@ const (
 	AttrScopeIdentity  = "identity"
 	AttrScopeSystem    = "system"
 	AttrScopeTags      = "tags"
-
-	AttrNameID       = "id"
-	AttrNameGroup    = "group"
-	AttrNameUpdated  = "updated_ts"
-	AttrNameCreated  = "created_ts"
-	AttrNameTagsEtag = "tags_etag"
+	AttrScopeMonitor   = "monitor"
+
+	AttrNameID             = "id"
+	AttrNameGroup          = "group"
+	AttrNameUpdated        = "updated_ts"
+	AttrNameCreated        = "created_ts"
+	AttrNameTagsEtag       = "tags_etag"
+	AttrNameNumberOfAlerts = "number_of_alerts"
```

Comment:
nit, feel free to ignore, why does `alert_count` sound better to me?

Reply:
as you wish ;)